modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -107,6 +107,11 @@
     #
     #liferay.workspace.plugins.sdk.dir=plugins-sdk
 
+    #
+    # Set the version of target platform. The default value is 1.0.0
+    #
+    #liferay.workspace.target.platform.version=1.0.0
```

Comment:
The default value should be 7.0.6 but we don't have official target platform BOM/Distro artifacts yet, so we can't release anything like this at the moment.  Please this property is not enabled by default (like all of the other properties in this file) so I'm not sure how we want to handle this.